internal/workflow_test.go

```diff
@@ -108,13 +108,17 @@ func TestConvertRetryPolicy(t *testing.T) {
 }
 
 func newTestWorkflowContext() Context {
-	return newWorkflowContext(&workflowEnvironmentImpl{
+	_, ctx, err := newWorkflowContext(&workflowEnvironmentImpl{
 		dataConverter: converter.GetDefaultDataConverter(),
 		workflowInfo: &WorkflowInfo{
 			Namespace:     "default",
 			TaskQueueName: "default",
 		},
-	}, nil, nil)
+	}, nil)
+	if err != nil {
```

Comment:
why not using assert?

Reply:
As an existing test helper function, I figured there was little value in changing existing callers to pass in a testing context for assertion and I also figured there was little value in changing existing callers to handle an error that should never happen. This is the simplest way for this test helper (I could just swallow the error I know will not happen, but we might as well panic to be future proof to other uses of the error).